package.json

```diff
@@ -17,10 +17,8 @@
 	},
 	"homepage": "https://github.com/osyrisrblx/rbx-datastore2/README.md",
 	"devDependencies": {
```

Comment:
Not too sure of the use of `peerDependencies` here. I think having `devDependencies` for `@rbxts/compiler-types` and `@rbxts/types` makes more sense here.

Reply:
There isn't?
You must have misread. I removed it as there was a use of `peerDependencies` but not anymore.

Reply:
Yep, LGTM!